packages/core/src/components/discord-messages/DiscordMessages.ts

```diff
@@ -78,6 +112,16 @@ export class DiscordMessages extends LitElement implements LightTheme {
 	@property({ type: Boolean, reflect: true, attribute: 'compact-mode' })
 	public accessor compactMode = false;
 
+	/**
+	 * The type of mention this should be. This will prepend the proper prefix character.
+	 * Valid values: `user`, `channel`, `role`, `voice`, `locked`, `thread`, `forum`, and `slash`.
+	 */
+	@property({ reflect: true, attribute: 'type' })
+	public accessor type: 'channel' | 'forum' | 'locked' | 'thread' | 'voice';
```

Comment:
```suggestion
	/**
	 * The type of channel this should be. This will prepend the proper prefix character.
	 * Valid values: `channel`, `forum`, `locked`, `thread`, and `voice`.
	 */
	@property({ reflect: true, attribute: 'type' })
	public accessor type: 'channel' | 'forum' | 'locked' | 'thread' | 'voice';
```